src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -40,29 +40,30 @@ public TimeSpan AutomaticRefreshInterval
         }
 
         /// <summary>
-        /// 12 hours is the default time interval that afterwards will obtain new configuration.
+        /// Default time interval (12 hours) after which a new configuration is obtained automatically.
         /// </summary>
         public static readonly TimeSpan DefaultAutomaticRefreshInterval = new TimeSpan(0, 12, 0, 0);
 
         /// <summary>
-        /// 1 hour is the default time interval that a last known good configuration will last for.
+        /// Default time interval (1 hour) for which the last known good configuration remains valid.
         /// </summary>
         public static readonly TimeSpan DefaultLastKnownGoodConfigurationLifetime = new TimeSpan(0, 1, 0, 0);
 
         /// <summary>
-        /// 5 minutes is the default time interval that must pass for <see cref="RequestRefresh"/> to obtain a new configuration.
+        /// Default time interval (5 minutes) that must pass before calling <see cref="RequestRefresh"/> to obtain a new configuration.
```

Comment:
There is a subtle point here.
One can call RequestRefresh as often as one wants, however, only after this time interval has passed will another call cause configuration to be obtained.

Reply:
I understand, thank you for the clarification! I've rollback `before calling` by `for`.